test/main/modules.js

```diff
@@ -63,6 +90,31 @@ QUnit.test( "runs after final test", function( assert ) {
 	assert.expect( 1 );
 } );
 
+var ranAfterHook;
+
+QUnit.module( "after (skip)", {
+	after: function( assert ) {
+		ranAfterHook = true;
+		assert.ok( true, "after hook ran" );
+	}
+} );
+
+QUnit.test( "does not run after initial tests", function( assert ) {
+	assert.expect( 0 );
+	ranAfterHook = false;
+} );
+
+QUnit.skip( "skipped, but should still run after hook", function( assert ) {
+	assert.expect( 1 );
+} );
+
+QUnit.module( "after (skip) verifier" );
+
+QUnit.test( "after hook ran after last (skipped) test", function( assert ) {
+	assert.expect( 1 );
+	assert.ok( ranAfterHook, "after hook ran" );
+} );
```

Comment:
can we nest the `after (skip)` and `after (skip) verifier` modules in a single module? mostly a

```js
QUnit.module( "after (skip)", function() {
	var ranAfterHook = 0;
	QUnit.module( "main", function(hooks) {
		hooks.after = function( assert ) {
			ranAfterHook += 1;
		};

		QUnit.test( "does not run after initial tests", ... );
		QUnit.skip( "skipped, but should still run after hook", ... );
	} );

	QUnit.module( "verifier", function() {
		QUnit.test( "after hook ran after last (skipped) test", ... );
	} );
} );
```